index.js

```diff
@@ -884,7 +885,7 @@ class GithubScm extends Scm {
             return branch.data.commit.sha;
         } catch (err) {
             logger.error('Failed to getCommitSha: ', err);
-            throw err;
+            throw new boom.Boom(err.message, { statusCode: err.status });
```

Comment:
We should use `boom.boomify` to keep information of error like the stack trace of it.
https://hapi.dev/module/boom/api/?v=9.1.1#boomifyerr-options

Reply:
Thanks. I fixed